Azure-Samples/aks-ray-sample
# Rdp migrate repo

## Purpose
Migrating code from private repo to public

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[X] Other... Please describe:
This is a new sample for deploying KubeRay on AKS and running a Ray application on the cluster.
```

## How to Test
*  Get the code

```
git clone[ [repo-address]](https://github.com/Azure-Samples/aks-ray-sample)
cd [repo-name]
git checkout [branch-name]
chmod +x ./deploy.sh
```

The script should complete and display the IP address to view the Ray dashboard on the AKS cluster


* ...

## Other Information
